web/blob/templates/_base.html

```diff
@@ -49,5 +49,7 @@
     </nav>
 
     {{template "content" .}}
+
+    {{ template "js" .}}
```

Comment:
Hmm, what's the benefit of templating the JS inclusion in this way here vs. just specifying it in the headers (if at all, then we'd also have to move the existing script tags in here that weren't moved in this PR).

If this is mainly about bottom-loading, I'm not sure if that makes a noticeable difference for Prometheus, but if so, we should just put all JS in all files at the bottom?

Otherwise I'd vote for just adding the one new JS file to the header in `status.html`.